src/components/buttons/DeleteButton.jsx

```diff
@@ -1,8 +1,8 @@
 import React, { PropTypes } from 'react';
-
 import IconButton from 'material-ui/IconButton';
 import TrashCan from 'material-ui/svg-icons/action/delete';
 import ReactTooltip from 'react-tooltip';
+import { branch, renderNothing } from 'recompose';
```

Comment:
The benefit is that it is established. At this point we all know what it means, and it's littered throughout the project. We shouldn't have two patterns to do the same thing.